lease/lessor.go

```diff
@@ -775,7 +792,7 @@ func (le *lessor) initAndRecover() {
 			revokec: make(chan struct{}),
 		}
 	}
-	heap.Init(&le.leaseHeap)
+	le.leaseExpiredNotifier.Init()
```

Comment:
There might be a bug in existing code. In this function `initAndRecover()`, leases are read from storage and written to leaseMap, but not to leaseHeap.

Reply:
Never mind. They are added later when lessor is promoted.